### PR TITLE
New package: git-remote-hg-1.0.2.1

### PR DESCRIPTION
**Pull request by Piraty**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/git-remote-hg/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+license="GPL-2.0-only"
+homepage="https://github.com/mnauw/git-remote-hg"
+distfiles="https://github.com/mnauw/git-remote-hg/archive/refs/tags/v${version}.tar.gz"
+checksum=a6af69ae22f31e86eb060636914088c8421e87f985ba18ae452fe6817aa74057
+python_version=3
+
+post_patch() {
+	vsed -i Makefile \
+		-e 's/prefix := $(HOME)/prefix := $(PREFIX)/'
+}
```
Comment (ericonr) [inline srcpkgs/git-remote-hg/template:19]: Hm, can't you pass `PREFIX=/usr` in `make_build_args`?

File: srcpkgs/git-remote-hg/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'git-remote'
+pkgname=git-remote-hg
+version=1.0.2.1
+revision=1
+build_style=gnu-makefile
+make_install_target="install install-doc"
+hostmakedepends="asciidoc"
+short_desc="Transparent bidirectional bridge between Git and Mercurial for Git"
```
Comment (ericonr) [inline srcpkgs/git-remote-hg/template:8]: I'm not clear on the relationship, but it's the official description, so I guess we should stick with it.

File: srcpkgs/git-remote-hg/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'git-remote'
```
Comment (ericonr) [inline srcpkgs/git-remote-hg/template:1]: And probably the xlint failure

File: srcpkgs/git-remote-hg/template (hunk 1 above)
Comment (Piraty) [reply srcpkgs/git-remote-hg/template:19]: `PREFIX` isn't used in any `install ...` commands in the makefile

Comment (ericonr) [reply srcpkgs/git-remote-hg/template:19]: sorry, I meant `prefix=/usr` instead.

**Pull request closed without merging**